packages/animated/src/AnimatedValue.ts

```diff
@@ -58,8 +56,10 @@ export class AnimatedValue<T = unknown> extends Animated
     if (is.num(this.value)) {
       this.startPosition = this.value
       this.lastPosition = this.value
-      this.lastVelocity = isActive ? this.lastVelocity : undefined
-      this.lastTime = isActive ? this.lastTime : undefined
+      if (!isActive) {
+        this.lastVelocity = null
```

Comment:
@aleclarson any reason why `this.lastVelocity` is reset to `null` and not `undefined`? `null` makes the test below not behave as it should:

```js
 velocity = animated.lastVelocity !== void 0 ? animated.lastVelocity : v0
```

Reply:
That was a mistake, but my local clone has `animated.lastVelocity == null` to account for that change.